### PR TITLE
Rename master branch to main 🧙

### PR DESCRIPTION
**Pull request by vdemeester**
Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

# Changes

This updates few reference of `master` to `main` as the default branch
is going to be `main` starting in the next few days.

/hold
/kind cleanup
To be merged once I renamed the branch to `main`.

/cc @tektoncd/cli-collaborators @tektoncd/cli-maintainers 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Run the code checkers with `make check`
- [x] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

File: RELEASE_PROCESS.md [hunk 1]
```diff
@@ -56,24 +56,24 @@ dnf upgrade tektoncd-cli
...
-    
+
     `72df3075303d2b0393bae9a0f9e9b8441060b8a4db57e613ba8f1bfda03809b5  cli-0.7.1.tar.gz`
 
   * Raise a PR to update like [this](https://github.com/Homebrew/homebrew-core/pull/46492) to Homebrew Core
 
 - Make a version update to the test-runner and `tkn` image in the [plumbing](https://github.com/tektoncd/plumbing/) repo. The test-runner image is used to run the CI on the pipeline project, which uses `tkn`. For both Dockerfiles listed below, search for `ARG TKN_VERSION` in the Dockerfile for where to update the release version. Update the version arg to match the version of `tkn` released (i.e. `ARG TKN_VERSION=<RELEASED_VERSION>`).
 
-  * [test-runner image](https://github.com/tektoncd/plumbing/blob/master/tekton/images/test-runner/Dockerfile) 
+  * [test-runner image](https://github.com/tektoncd/plumbing/blob/cli/tekton/images/test-runner/Dockerfile)
```
Comment (vinamra28) [inline RELEASE_PROCESS.md:72]:
```suggestion
  * [test-runner image](https://github.com/tektoncd/plumbing/blob/main/tekton/images/test-runner/Dockerfile)
```

File: RELEASE_PROCESS.md [hunk 2]
```diff
@@ -56,24 +56,24 @@ dnf upgrade tektoncd-cli
...
 
-  * [tkn image](https://github.com/tektoncd/plumbing/blob/master/tekton/images/tkn/Dockerfile)
+  * [tkn image](https://github.com/tektoncd/plumbing/blob/cli/tekton/images/tkn/Dockerfile)
```
Comment (vinamra28) [inline RELEASE_PROCESS.md:74]:
```suggestion
  * [tkn image](https://github.com/tektoncd/plumbing/blob/main/tekton/images/tkn/Dockerfile)
```

**Review by vinamra28 — CHANGES_REQUESTED** [review]

File: RELEASE_PROCESS.md (hunk 1 above)
Comment (vdemeester) [reply RELEASE_PROCESS.md:72]: oh lol 😹

**Comment by chmouel** [comment]
let there be the dragons

/lgtm

**Review by pradeepitm12 — APPROVED** [review]

**Comment by vdemeester** [comment]
hum 🤔

**Comment by vdemeester** [comment]
Retrying to see if those are flake or legit…
/retest

**Comment by vdemeester** [comment]
```
tektoncd           3m26s       Warning   FailedCreate                   replicaset/el-github-listener-interceptor-64786d4d49                 Error creating: pods "el-github-listener-interceptor-64786d4d49-" is forbidden: error looking up service account tektoncd/tekton-triggers-github-sa: serviceaccount "tekton-triggers-github-sa" not found
```

Is this a race ? or do we miss something ? 🤔

**Comment by vinamra28** [comment]
this is what I am trying to see by doing a retest. I just saw that there was one PR related to eventlistener failure

**Comment by barthy1** [comment]
based on my tests with `master`, the error was different - 
```
k8s.io/client-go@v0.19.7/tools/cache/reflector.go:156: Failed to watch *v1alpha1.TriggerTemplate: failed to list *v1alpha1.TriggerTemplate: triggertemplates.triggers.tekton.dev is forbidden: User "system:serviceaccount:testcli-67zcj:tekton-triggers-github-sa" cannot list resource "triggertemplates" in API group "triggers.tekton.dev" in the namespace "testcli-67zcj
```

Yesterday the error `looking up service account tektoncd/tekton-triggers-github-sa: serviceaccount "tekton-triggers-github-sa" not found` didn't exist.

**Comment by vinamra28** [comment]
also I just did try out the e2e tests locally on my machine for EventListener and they are passing

**Comment by barthy1** [comment]
looks like problem with cluster setup? I can see in the error logs of the tests
```
default            105s        Normal    VolumeDelete                   persistentvolume/pvc-def086e6-a100-47c9-bcb0-5ab6e84767c4            googleapi: Error 400: The disk resource 'projects/tekton-prow-11/zones/us-central1-a/disks/gke-tcli-e2e-cls136933-pvc-def086e6-a100-47c9-bcb0-5ab6e84767c4' is already being used by 'projects/tekton-prow-11/zones/us-central1-a/instances/gke-tcli-e2e-cls13693307-default-pool-bc7a78fa-40qh', resourceInUseByAnotherResource
... skipping 115 lines ...
kube-system        6m45s       Warning   FailedCreate                   replicaset/kube-dns-57fcf698d8                                       Error creating: insufficient quota to match these scopes: [{PriorityClass In [system-node-critical system-cluster-critical]}]
... skipping 11 lines ...
kube-system        6m45s       Warning   FailedCreate                   replicaset/kube-dns-autoscaler-7f89fb6b79                            Error creating: pods "kube-dns-autoscaler-7f89fb6b79-" is forbidden: error looking up service account kube-system/kube-dns-autoscaler: serviceaccount "kube-dns-autoscaler" not found
... skipping 47 lines ...
kube-system        6m45s       Warning   FailedCreate                   replicaset/metrics-server-v0.3.6-8497fd5d6d                          Error creating: insufficient quota to match these scopes: [{PriorityClass In [system-node-critical system-cluster-critical]}]
... skipping 51 lines ...
kube-system        6m44s       Warning   FailedCreate                   replicaset/stackdriver-metadata-agent-cluster-level-569b7b5fb6       Error creating: insufficient quota to match these scopes: [{PriorityClass In [system-node-critical system-cluster-critical]}]
... skipping 24 lines ...

```

**Comment by barthy1** [comment]
@vdemeester are you sure you want to delete these resources https://github.com/tektoncd/cli/pull/1316/files#diff-1cbb8009d3d7b4d91bd3f67ca822853185dfe921d7752f44a4408551207bab29L84 ? it looks like reason of the test error you have

**Comment by barthy1** [comment]
oops, sorry, now I see that it's just some kind of reformatting. Bit still looks related to the error...

**Comment by vdemeester** [comment]
Hum…

```
cat: '"/tmp/tmp.CuCk2OKapG/changed_files"': No such file or directory
==========================================='========================================
==== NO CHANGED FILES REPORTED, ASSUMING IT'S AN ERROR AND RUNNING TESTS ANYWAY ====
==========================================='========================================
```

This doesn't seem right, I need to check if there is a problem here.

File: test/e2e/clustertask/start_test.go [hunk 3]
```diff
@@ -43,6 +43,7 @@ const (
 )
 
 func TestClusterTaskInteractiveStartE2E(t *testing.T) {
+	t.Skip("TEMPORARY DISABLING THIS, SEE https://github.com/tektoncd/cli/issues/1319.")
```
Comment (chmouel) [inline test/e2e/clustertask/start_test.go:46]: is it temporary? :trollface:

**Review by chmouel — COMMENTED** [review]
/lgtm

Looking Good! 🤙🏽

**Pull request merged**